docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
 
-![Registration network with weak supervision loss.\label{fig:weakly}](https://github.com/DeepRegNet/DeepReg/blob/master/docs/source/_images/registration-ddf-nn-weakly-supervised.svg?raw=true)
+![Registration network with weak supervision loss.\label{fig:weakly}](../source/_images/registration-ddf-nn-weakly-supervised.png)
 
 ## Combining unsupervised loss with weak supervision
 Combining the unsupervised loss and the weak supervision has shown superior registration accuracy, compared with that using unsupervised loss alone [@balakrishnan2019voxelmorph]. As a result, the overall loss is the weighted sum of the image dissimilarity, the label dissimilarity and the deformation regularisation, which constrains smoothness of the predicted transformation and is described in the next section. \autoref{fig:combined} shows a schematic illustration of the weakly-supervised network training with unsupervised loss.
 
-![Registration network with combined unsupervised and weakly-supervised losses.\label{fig:combined}](https://github.com/DeepRegNet/DeepReg/blob/master/docs/source/_images/registration-ddf-nn-combined.svg?raw=true)
+![Registration network with combined unsupervised and weakly-supervised losses.\label{fig:combined}](../source/_images/registration-ddf-nn-combined.png)
 
 ## Deformation regularisation and conditional segmentation
-The loss functions described in the previous sections are often combined with a deformation regularisation term on the predicted displacement field, in order to ensure the predicted deformation is smooth. The deformation can be regularised by L1-, L2 norms of the displacement gradient and bending energy, based on the first- or second derivatives of DDF.
+The loss functions described in the previous sections are often combined with a deformation regularisation term on the predicted displacement field, in order to ensure the predicted deformation is smooth. The deformation can be regularised by L1-, L2 norms of the displacement gradient and bending energy, based on the first- or second derivatives of the predicted DDF. Primarily predicting a general dense displacement field (DDF) from the network (discussed further in the next section), other parameterised transformation models can be readily added to the DeepReg framework, such as rigid transformation model and free-form deformation model based on B-splines [@rueckert1999nonrigid], to further constrain the predicted transformation. As an example, DeepReg implements an affine transformation model with twelve degrees of freedom.
```

Comment:
Here "predicted DDF" is the first time we use DDF, better to move the dense displacement field forward

Reply:
"free-form deformation" free-form or form-free ?

Reply:
free-form - see the cited paper

Reply:
although form-free sounds cool :D